sacrebleu/__init__.py

```diff
@@ -14,7 +14,7 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-__version__ = '1.4.14'
+__version__ = '1.5'
 __description__ = 'Hassle-free computation of shareable, comparable, and reproducible BLEU, chrF, and TER scores'
```

Comment:
Should this be 1.5.0? Or maybe trailing zeros are left off, not sure.

Reply:
done